src/main/steps/c100-rebuild/child-details/child-matters/content.ts

```diff
@@ -98,9 +167,14 @@ export const form: FormContent = {
 
 export const generateContent: TranslationFn = content => {
   const translations = languages[content.language]();
+  const sessionData = content?.userCase ?? {};
   const childId = content.additionalData!.req.params.childId;
   const childDetails = getChildDetails(content.userCase!.cd_children ?? [], childId)!;
-  const { fields } = generateFormFields(childDetails.childMatters);
+  const { fields } = generateFormFields(
+    childDetails.childMatters ?? getDataShape().childMatters,
```

Comment:
@giorgi-hmcts We need to cal getChildDetails here not getDataShape

Reply:
Fixed as per the comment.